### PR TITLE
Multiline gl2d texts

### PR DESCRIPTION
**Pull request by monfera**
PR depends on the merge and npm publish of its [gl-vis dependency](https://github.com/gl-vis/gl-plot2d/pull/6) 

Purports to fix https://github.com/plotly/plotly.js/issues/1079

### CODE REVIEW

File: src/plots/gl2d/scene2d.js [hunk 1]
```diff
@@ -249,9 +249,7 @@ proto.computeTickMarks = function() {
 
     for(var j = 0; j < 2; ++j) {
         for(var i = 0; i < nextTicks[j].length; ++i) {
-            // TODO add support for '\n' in gl-plot2d,
-            // For now, replace '\n' with ' '
-            nextTicks[j][i].text = convertHTMLToUnicode(nextTicks[j][i].text + '').replace(/\n/g, ' ');
+            nextTicks[j][i].text = convertHTMLToUnicode(nextTicks[j][i].text + '');
```
Comment (etpinard) [inline src/plots/gl2d/scene2d.js]: why the extra `+ ''` ?

**Comment by monfera** [comment]
It was there and I left it intact, thinking it was in place to coerce it into a string

File: src/plots/gl2d/scene2d.js (hunk 1 above)
Comment (monfera) [inline src/plots/gl2d/scene2d.js]: It was there and I left it intact, thinking it was in place to coerce it into a string

Comment (etpinard) [inline src/plots/gl2d/scene2d.js]: I believe `nextTicks[j][i].text` will always be a string. Let's :hocho: that `+ ''`.

Comment (monfera) [inline src/plots/gl2d/scene2d.js]: OK

Comment (etpinard) [inline src/plots/gl2d/scene2d.js]: Ha.  Good to know. Revert the :hocho: and add a comment about the `+ ''`.

**Comment by etpinard** [comment]
Awesome work @monfera 

Thanks very much!

**Pull request merged**